roles/oneview_server_profile/meta/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+galaxy_info:
+  role_name: oneview_server_profile
+  author: Chebrolu Bala Sai Harika
+  description: Ansible role to manage OneView Fibre Channel Network resources.
```

Comment:
it should be as below. Should mention this resource name.
```
  description: Ansible role to manage OneView Server Profile resources.
```

Reply:
done